packages/front-end/components/Experiment/VariationsTable.tsx

```diff
@@ -7,6 +7,9 @@ import { useAuth } from "@/services/auth";
 import { trafficSplitPercentages } from "@/services/utils";
 import Carousel from "../Carousel";
 import ScreenshotUpload from "../EditExperiment/ScreenshotUpload";
+import AuthorizedImage from "../AuthorizedImage";
+
+const imageCache = {};
```

Comment:
This might be better suited to be defined by a useState variable, and reset when `variation.id` changes

```
const [imageCache, setImageCache] = useState({});

useEffect(() => {
  setImageCache({});
}, [variation.id]);

// you can still set imageCache as it's currently done
imageCache[src] = blob;
```

Reply:
Doing that would make the cache disappear as they navigate to other pages and come back.  Keeping it as is makes the cache last longer.